e2e-cypress/Jenkinsfile.template

```diff
@@ -29,6 +29,17 @@ odsComponentPipeline(
     // 'release/': 'test'
   ]
 ) { context ->
+
+  // Note: Testing in the production environment is not recommended as it can lead to unintended consequences,
+  // including potential downtime, data corruption, or exposure of sensitive information.
+  // This block is designed to skip tests in the production environment to avoid these risks.
+  // If you choose to enable testing in production, do so at your own risk and take all necessary precautions.
+  if (context.environment == 'prod') {
```

Comment:
@roicarrera the Cypress Quickstarter is of type ods-test, which must support acceptance, integration, and installation tests. You cannot simply omit all tests in production. Since installation tests are mandatory by LeVA in the PROD environment, you are right now messing with compliance regulations. Changing the current testing behavior in PROD needs a clear design in agreemtn with the core platform team and consistency across all ods-test Quickstarters. FYI @BraisVQ

Reply:
@roicarrera your comment says that testing in production means to take a risk and requires precaution. What do you mean? We made this conciously to ensure that applications, once in production, work as expected. Limiting test execution to installation tests is a possible and helpful improvement for the future for sure.